docs/tutorial/node/src/getting_started.md

```diff
@@ -0,0 +1,44 @@
+# Getting Started
+
+In this tutorial, we use JavaScript as the host programming language. We also support other programming languages like
+[Rust](https://slint-ui.com/docs/rust/slint/) or [C++](https://slint-ui.com/docs/cpp/).
+
+You will need a development environment with [node.js 16](https://nodejs.org/download/release/v16.19.1/) and [npm](https://www.npmjs.com/) installed.
+
+We're going to use `slint-ui` as `npm` dependency.
+
+In a new directory, we create a new `package.json` file.
+
+```json
+{{#include package.json}}
+```
+
+This should look familiar to people familiar with nodejs. We see that this package.json
+references a `main.js`, which we will add later. We must then create, in the same directory,
+the `memory.slint` file. Let's just fill it with a hello world for now:
+
+```slint
+{{#include memory.slint:main_window}}
+```
+
+What's still missing is the `main.js`:
+
+```js
+{{#include main_initial.js:main}}
+```
+
+To recap, we now have a directory with a `package.json`, `memory.slint` and `main.js`.
+
+We can now compile and run the program:
+
+```sh
+npm install
+npm start
+```
```

Comment:
did you try that this actually works?

I think peple had issue with that and had to run npm install somewhere esle too.  Or was that fixed?

Reply:
Yes we no trouble.